plugin/release.go

```diff
@@ -50,30 +50,61 @@ func (rc *releaseClient) buildRelease() (*github.RepositoryRelease, error) {
 }
 
 func (rc *releaseClient) getRelease() (*github.RepositoryRelease, error) {
-	release, _, err := rc.Client.Repositories.GetReleaseByTag(rc.Context, rc.Owner, rc.Repo, rc.Tag)
 
-	if err != nil {
-		return nil, fmt.Errorf("release %s not found", rc.Tag)
-	}
+	listOpts := &github.ListOptions{PerPage: 10}
```

Comment:
Why only 10?

Reply:
I'm fine myself with whatever number we want to choose here. I just took it from @donny-dont's code he linked before.
However, I also noted this under "Possible Enhancements" in the PR description.
Which number would you choose?

Reply:
Probably GitHub API default 30 would be better imho

Reply:
In regular case it would anyway find the release on the first page, let's ignore my comment and merge it.